### ISSUE TITLE
CI not working

### ISSUE THREAD

**Issue by asinghvi17** [JuliaPlots/Makie.jl#296]
Gitlab CI gives this error:

```bash
$ julia -e 'using InteractiveUtils; versioninfo()'
Julia Version 1.0.3
Commit 099e826241 (2018-12-18 01:34 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2637 v2 @ 3.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, ivybridge)
Environment:
  JULIA_DEPOT_PATH = /builds/JuliaGPU/Makie.jl/.julia/
$ julia --project=test/ -e 'using Pkg; Pkg.build("Makie"); include("test/runtests.jl")'
ERROR: The following package names could not be resolved:
 * Makie (not found in project or manifest)
Please specify by known `name=uuid`.
Stacktrace:
 [1] pkgerror(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Types.jl:120
 [2] #ensure_resolved#43(::Bool, ::Function, ::Pkg.Types.EnvCache, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Types.jl:900
 [3] ensure_resolved at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Types.jl:870 [inlined]
 [4] #build#53(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:455
 [5] build at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:436 [inlined]
 [6] build(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:431
 [7] top-level scope at none:0
ERROR: Job failed: exit code 1
```

This is a subset of the full output.  It seems that there's an error where Makie isn't being found by Julia, maybe the directory is wrong?

**Comment by asinghvi17** [comment]
See JuliaPlots/Makie.jl@8e9fb1ed3cdb6a565914bb2c825117e170fcdd0e

**Comment by asinghvi17** [comment]
This should be fixed in a while by #297
